pkgs/development/python-modules/flit/default.nix

```diff
@@ -38,6 +40,8 @@ buildPythonPackage rec {
     requests
     requests_download
```

Comment:
That is no longer required https://github.com/pypa/flit/blob/3.6.0/pyproject.toml

```suggestion
```